Docs: Time series panel (#30690)
grafana/grafana: docs/sources/panels/visualizations/time-series/graph-time-series-as-points.md

1. In the [Panel editor](https://grafana.com/docs/grafana/latest/panels/panel-editor/), click the **Field** tab.
1. In Style, click **Points**.

## Style the points

Use the following field settings to refine your visualization.

For more information about applying these options, refer to:

- [Configure all fields]({{< relref "../../field-options/configure-all-fields.md" >}})
- [Configure specific fields]({{< relref "../../field-options/configure-specific-fields.md" >}})

Some field options will not affect the visualization until you click outside of the field option box you are editing or press Enter.

### Point size

Set the size of the points, from 1 to 40 pixels in diameter.

Point size set to 6:

![Show points point size 6 example](/img/docs/time-series-panel/points-graph-show-points-6-7-4.png)

Point size set to 20:

![Show points point size 20 example](/img/docs/time-series-panel/points-graph-show-points-20-7-4.png)

Point size set to 35:

![Show points point size 35 example](/img/docs/time-series-panel/points-graph-show-points-35-7-4.png)

## Scatter plot graph examples

> Below are some scatter plot examples to give you ideas.

Reviewer: ```suggestion
Some scatter plot examples can give you ideas.
```

> DOMINIK OR TORKEL - GOT ANY EXAMPLES?

Reviewer: ```suggestion
```
